Aquilon initial configuration: fix template examples

jouvin opened the pull request:
* `archetype/final.pan`: add missing Quattor client configuration
* `archetype/base.pan`: add a file system configuration example

ned21 commented on _aquilon/00-install.md:
```diff
@@ -509,6 +516,26 @@ the main configuration file is:
+AII (deployment) server can be run on the same machine as the Aquilon broker or on a separate machine.
+In both cases, this requires the broker to be able to connect the AII server through SSH using the
+account specified by configuration option `installfe_user`(the broker account name by default) and, from
+this account, to execute the command `aii-shellfe` as root trough `sudo`.
```
trough => through

ned21 commented on _aquilon/00-install.md:
```diff
@@ -509,6 +516,26 @@ the main configuration file is:
+Main steps involved are:
+* If the AII server is not running on the broker host, on the AII server create the account matching 
+`installfe_user` configuration option (the broker account name by default).
+* On the AII server, configure `sudo` to allow the created account to execute `usr/sbin/shellfe` as root
```
Is `usr/sbin/shellfe` correct or should it be `/usr/sbin/shellfe` ?

ned21 commented on _aquilon/management.md:
````diff
@@ -238,6 +238,117 @@ If you need an external network, you have to create it with
+all the defined networks):
+
+```bash
+aq map service --service dns --instance metropolis1  --networkid 192.168.1.0
````
Using network maps are not very flexible because unlike locations you can't take advantage of the hierarchy. If we are creating an example config I think it should demonstrate the full features of what Aquilon has to offer, i.e. multi-level maps. Would it be possible to change this to be based on a building or city instead of a network?

(tbh, network maps were added as a bit of a hack to account for DMZs and special enclaves which defied geographic categorisation. In practice they cause us no end of pain.)

ned21 commented on _aquilon/management.md:
````diff
@@ -238,6 +238,117 @@ If you need an external network, you have to create it with
+To bind one host, use:
+
+```bash
+aq bind client --service dns --hostname preprod.dailyplanet.com
````
You should never run bind_client. This is for temporary bindings, and will get removed when you run `aq reconfigure` which reevaluates all existing bindings against the maps. First you need to define the service as required as per the next step.

ned21 commented on _aquilon/management.md:
````diff
@@ -238,6 +238,117 @@ If you need an external network, you have to create it with
+aq add required service --service dns --archetype web_servers
+```
+
+Note that the `aq bind client` command (or `aq reconfigure`) will fail if there is
````
Delete reference to bind_client. Clients are always bound via `aq reconfigure`

ned21 commented on _aquilon/management.md:
````diff
@@ -238,6 +238,117 @@ If you need an external network, you have to create it with
+To unbind a client from a service instance, use:
+
+```bash
+aq unbind client --service dns --hostname preprod.dailyplanet.com
````
You need to remove the map/required service and then reconfigure. Without this, `aq reconfigure` will put this back.

jouvin commented:
@ned21 thanks for your careful review. Hopefully the new version addresses your remarks. Am I right saying that a host can be assigned to a service only by marking the service as required for the archetype/personality or did I miss something?
